charts/registry/templates/tests/test-connection.yaml

```diff
@@ -17,10 +17,15 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 ###############################################################
+
+{{- $fullName := include "chart.fullname" . }}
+{{- $testconnection_name := printf "%s-test-script-pod" $fullName }}
+{{- $testsec_name := printf "%s-test-credentials" $fullName }}
+{{- $testconfig_name := printf "%s-test-script" $fullName }}
```

Comment:
I would handle that the same way as in the other templates. But it's up to you if you want to keep it consistent or not 😉